playroom/frame-component.tsx

```diff
@@ -31,7 +31,7 @@ const App: React.FC = ({children}) => {
 
         ${isModalOpen ? 'body {overflow-y: hidden}' : ''}
 
-        *[class^='SplashScreen__'] {
+        *[class^='_'] {
```

Comment:
@atabel this solve the splashscreen.. but.. maybe is too generic? I don't know if this one could cause problems in other places

Reply:
yes, this is too generic and may bite us